curvefs/src/client/metric/client_metric.h

```diff
@@ -275,18 +276,32 @@ struct S3Metric {
           writeSize(prefix, fsName + "_adaptor_write_size", 0) {}
 };
 
+template <typename Tp>
+uint64_t atomValue2Bytes(void* atomValue) {
```

Comment:
`LoadAtomicValue`?

and why return `uint64_t` while these variables are `int64_t`?

Reply:
> `LoadAtomicValue`?
> 
> and why return `uint64_t` while these variables are `int64_t`?

fix